node/src/Pos/Generator/Block/Payload.hs

```diff
@@ -212,7 +211,7 @@ genTxPayload = do
 
         eTx <- lift . lift $
             createGenericTx makeTestTx ownUtxo txOutAuxs changeAddrData
-        (txAux, _) <- either (throwM . BGFailedToCreate . unTxError) pure eTx
+        (txAux, _) <- either (throwM . BGFailedToCreate . pretty) pure eTx
```

Comment:
I think it would be better to put `TxError` into `BGFailedToCreate` instead, but it's minor.

Reply:
There are two instances where `BGFailedToCreate` is used, so I think it will be easier to leave this as-is now.